lib/collections.js

```diff
@@ -17,6 +17,20 @@ var normalizedWords = [];
 
 /* your code starts here */
 
+normalizedWords = paragraph.split(' ');
+
+for (var i = 0; i < normalizedWords.length; i++) {
+  normalizedWords[i] = normalizedWords[i].replace(/[^\w']+/, '').toLowerCase();
+}
+
+/* OR */
+/*
+normalizedWords = normalizedWords.map(function(word) {
+  return word.replace(/[^\w']+/, "");
+});
```

Comment:
Yep! Just don't forget to do `toLowerCase`. 

Actually, if you want to get fancy, you could even write

```javascript
normalizedWords = paragraph.split(' ').map(function(word){
  return word.replace(/[^\w']+/, '').toLowerCase();
}); 
```